### PR TITLE
Added Project Files

### PR DESCRIPTION
**Pull request by Divyan-H**
Team, kindly review my code and provide necessary feedback for improvements. Your suggestions are valuable in enhancing its quality. Thank you for your assistance in making it even better.

Screenshot from 2023-07-11 14-07-58

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+    </div> 
+    <!--Navigation Bar Ends-->
+    <img src="https://files.auroville.org/auroville-org/cd2883ee-eef9-4094-8ba2-29f5da05b0c1.jpg" 
+    alt="space-img"
+    id="jumbotron">
+    <div id="description">
+        <i>There should be somewhere on earth a place which no nation could claim as its own, 
+        where all human beings of goodwill who have a sincere aspiration could live freely 
+        as citizens of the world and obey one single authority, that of the supreme Truth.</i>
+        <div id="the-mother">The Mother</div>
```
Comment (sangeethailango) [inline index.html]: Indent the whole code properly.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+            alt="Matrimandir"
+            id="matrimandir">
+            <div class="container-2">
+                <h4>3 Core Documents</h4>
+                <p>The Auroville Charter, A Dream and To Be A True
+                    Aurovillian - these 3 sentimental documents define the
+                    essence of auroville
+                </p>
+            </div>
+        </div>
```
Comment (sangeethailango) [inline index.html:171]: Good indentation.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+                    <p>Auroville was founded by The Mother (Mirra Alfassa)
+                       who was the spiritual collabrator of the Indian Sage
+                       Sri Aurobindo
+                    </p>
+                </div>
+            </div>
+        </a>
+        <a href="https://auroville.org/page/integral-yoga">
+            <div class="cards">
+                <img src="https://files.auroville.org/auroville-org/a21d0974-55b2-4ab3-ab62-ced4db9c90be.jpg"
```
Comment (sangeethailango) [inline index.html]: Use relative file paths for all of the images which is in this page. Read the below documentation on file paths.
https://www.w3schools.com/html/html_filepaths.asp

**Review by sangeethailango — CHANGES_REQUESTED** [review]

**Review by sakthibalan15 — APPROVED** [review]

**Review by nittin-shankar — APPROVED** [review]
If you do all the changes that @sangeethailango has given, you're good to merge it.

**Pull request merged**